cli/tests/unit/text_encoding_test.ts

```diff
@@ -50,6 +50,16 @@ unitTest(function atobThrows2(): void {
   assert(threw);
 });
 
+unitTest(function atobThrows3(): void {
+  let threw = false;
+  try {
+    atob("foobar!!");
+  } catch (DOMException) {
```

Comment:
This unit test needs to:

- Check that the caught error is an instance of DOMException
- Validate that the `toString()` of the caught error starts with the text `DOMException:`

Reply:
When I execute the following on chrome, I get this error.
```
try {
    atob("foobar!!");
} catch (e) {
    console.log(e.toString())
}
```
InvalidCharacterError: Failed to execute 'atob' on 'Window': The string to be decoded is not correctly encoded.

What should I do?

Reply:
Hi! @gorogoroumaru, @kitsonk 

What about the following approach?

```javascript
let threw = false;
try {
  ...
} catch (e) {
  if (e instanceof DOMException) {
    threw = true;
  }
}
assert(threw);
```